mods/rv/rules/ai-only-units.yaml

```diff
@@ -0,0 +1,229 @@
+### This file defines units can be only built by AI player, which
+### 1. will use diffirent set of logic compared to orignal units used by AI
+### 2. are pre-loaded transprot units with unload logic.
+
+fv-e1:
+	Inherits: fv
+	Buildable:
+		BuildPaletteOrder: 31
+		Prerequisites: ~prerequisite.aiplayer, ~techlevel.1
+	ProductionCostMultiplier:
+		Multiplier: 125
```

Comment:
When i added these units, i made it so both build time and cost equals to build time and cost of the vehicle plus its contents. We make the AI cheat but does it really need that much? I say we should keep it that way.

Reply:
Because 
1. AI cannot use many of the units well (like chrono-IFV), 
2. some cheat for Allied is to make Allied can last to late game when against Soviet AI, while I am doing AI test (but Allied AI cannot beat Soviet AI in all of my tests even that)

So it can be adjust if you like, but I suggest in later PR